configure

```diff
@@ -1513,6 +1518,10 @@ pprint.pprint(output, indent=2)
 write('config.gypi', do_not_edit +
       pprint.pformat(output, indent=2) + '\n')
 
+write('config.status', '#!/bin/sh\nset -ex\n./configure ' +
+      ' '.join([shell_quote(arg) for arg in original_argv]) + '\n')
```

Comment:
Use `pipes.quote(arg)` for this (officially deprecated but `shlex.quote` doesn't exist in python 2.7.)

Reply:
Done!